cli_tools/diagnostics/main_windows_test.go

```diff
@@ -0,0 +1,303 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// These test can only be run on windows, as the functions are highly dependent on windows OS.
+package main
+
+import (
+	"errors"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func Test_cmd_run(t *testing.T) {
+	type fields struct {
+		path            string
+		args            string
+		outputFileName  string
+		cmdProducesFile bool
+	}
+	tests := []struct {
+		name        string
+		fields      fields
+		wantOutPath string
+		wantErr     bool
+	}{
+		// TODO: Add test cases.
```

Comment:
a bit confused, is this a incomplete PR?

Reply:
removed the unnecessary test template